core/src/main/java/com/ritense/valtimo/web/rest/dto/CustomTaskDto.java

```diff
@@ -29,8 +30,10 @@ public class CustomTaskDto implements Serializable {
 
     private CamundaTaskDto task;
 
+    @Transient
```

Comment:
Sonar has a problem with this fields in a Serialized class  that has no @Transcient or @Serializable annotation. Sonar says that : it could cause program crashes  or is a security risk while opening the door to attackers. No problem here to roll this change back.